doandroidlib/src/main/java/in/tosc/doandroidlib/DigitalOcean.java

```diff
@@ -54,9 +54,9 @@ public static String getCallbackUrl() {
 
     static String callbackUrl;
     static String authToken = "";
-    static Retrofit r, r2;
+    static Retrofit r, r2, r3;
```

Comment:
yeah, we can, actually its not needed will remove this whole block and fix it ASAP